pyworklistserver/user_config.py

```diff
@@ -0,0 +1,51 @@
+""" Sets variables for use of the worklist-generators"""
+import random
+
+#Maximall number of total patient-objects delivered from generator.
+maxAmountOfWorklistExams = 70
+
+#Minimum number of total patient-objects delivered from generator.
+minAmountOfWorklistExams = 20
+
+#Rate of clean patient-objects delivered from generator.
+rateOfCleanExams = 2
+
+#Rate of random patient-objects delivered from generator.
+rateOfRandomExams = 5
+
+#Generator seed value, if command line argument not set. Set to 0 for seed to be set to a random number.
+seed_Number = 0
```

Comment:
Instead, should be the internal state of the random number generator per C-FIND request.

Reply:
Can you elaborate on what the issue is and how to fix it?